share/spice/pokemon/data/pokemon_data.js

```diff
@@ -0,0 +1,119 @@
+(function (env) {
+    "use strict";
+
+    var ID = 'pokemon';
+
+    /**
+     *  [ Pokemon::Data ]
+     */
+    env.ddg_spice_pokemon_data = function(api_result){
+        var DESCRIPTION_ENDPOINT = '/js/spice/pokemon/description/{id}',
+            POKEAPI_SPRITE_URL = 'http://pokeapi.co/media/img/{id}.png',
+            INFOBOX_PROPS = ['hp', 'attack', 'defense', 'height', 'weight', 'speed'];
+
+        if (!api_result) {
+            return Spice.failed('pokemon');
+        }
+
+        Spice.add({
+            id: ID,
+            name: 'Pokedex',
+            data: api_result,
+            meta: {
+                sourceName: 'pokeapi.co',
+                sourceUrl: 'http://pokeapi.co/',
+                sourceIconUrl: 'http://pokeapi.co/static/favicon.ico'
+            },
+            normalize: function(item) {
+                return {
+                    title: item.name,
+                    image: getSprite.call(item),
+                    infoboxData: getInfoboxData.call(item),
+                    eggGroups: getCollectionNames.call(item, 'egg_groups'),
+                    types: getCollectionNames.call(item, 'types')
+                };
+            },
+            templates: {
+                group: 'info',
+                options: {
+                    content: Spice.pokemon_data.content,
+                    moreAt: true
+                }
+            }
+        });
+
+        if( api_result.descriptions.length > 0 ) {
```

Comment:
@AkA84 thanks a lot! I definitely understand your reasoning, but I have a suggestion, so that we can remove the `Pokemon::Description` callback! Will comment on that line...